concepts/05 UI Components/TagBox/00 Getting Started with TagBox/00 Getting Started with TagBox.md

```diff
@@ -0,0 +1,9 @@
+TagBox is an editor that allows users to select multiple items from a drop-down list.
+
+This tutorial explains how to add a TagBox to a page, populate its drop-down list, allow users to search through it, and log the selected items data to the console.
```

Comment:
This is also a very long and strange sentence. You definitely need to rewrite this. Please, take a look:

> ... add a TagBox to **a page**, populate **its** drop-down list

While it's seemingly obvious that you populate the tag box's drop-down list, the pronoun "its" points at "a page".

> log the selected items data to the console.

What does it even mean? To log the data of selected items to the console? If yes, you should've written:

> and log selected item data to the console.